### PR TITLE
[MTA-2660] Updating installation instructions

### PR DESCRIPTION
**Pull request by mpershina**
### Goal:

Update MTA [Installation instructions](https://access.redhat.com/documentation/en-us/migration_toolkit_for_applications/7.0/html-single/user_interface_guide/index#mta-7-installing-web-console-on-openshift_user-interface-guide) to include all OpenShift editions, both [cloud](https://www.redhat.com/en/technologies/cloud-computing/openshift#cloud-services-editions) and [self managed](https://www.redhat.com/en/technologies/cloud-computing/openshift#self-managed).

### CODE REVIEW

File: docs/topics/mta-7-installing-web-console-on-openshift.adoc [hunk 1]
```diff
@@ -7,9 +7,9 @@
 
 = Installing the {ProductName} {WebName}
 
-You can install the {ProductName} ({ProductShortName}) {WebName} as part of the process of installing the {ProductShortName} Operator on the OpenShift Container Platform.
+You can install the {ProductName} ({ProductShortName}) {WebName} on all cloud services and self-managed OpenShift editions.
```
Comment (mpershina) [inline docs/topics/mta-7-installing-web-console-on-openshift.adoc:10]:
```suggestion
You can install the {ProductName} ({ProductShortName}) {WebName} on all OpenShift cloud services and self-managed OpenShift editions.
```

File: docs/topics/mta-7-installing-web-console-on-openshift.adoc [hunk 2]
```diff
@@ -52,19 +52,19 @@ To successfully deploy, the {ProductShortName} Operator requires 3 RWO persisten
 
 == Installing the {ProductName} Operator and the {WebName}
 
-You can install the {ProductName} ({ProductShortName}) and the {WebName} on OpenShift Container Platform versions 4.13-4.15 when you install the {ProductName} Operator.
+You can install the {ProductName} ({ProductShortName}) and the {WebName} on OpenShift versions 4.13-4.15 when you install the {ProductName} Operator.
```
Comment (mpershina) [inline docs/topics/mta-7-installing-web-console-on-openshift.adoc:55]:
```suggestion
You can install the {ProductName} ({ProductShortName}) and the {WebName} on Red Hat OpenShift versions 4.13-4.15 when you install the {ProductName} Operator.
```

File: docs/topics/mta-7-installing-web-console-on-openshift.adoc [hunk 3]
```diff
@@ -52,19 +52,19 @@ To successfully deploy, the {ProductShortName} Operator requires 3 RWO persisten
...
 
 .Prerequisites
 
 * 4 vCPUs, 8 GiB RAM, and 40 GiB persistent storage.
-* OpenShift Container Platform 4.13-4.15 installed.
+* Any cloud services or self- hosted edition of OpenShift on versions 4.13-4.15.
```
Comment (mpershina) [inline docs/topics/mta-7-installing-web-console-on-openshift.adoc:60]:
```suggestion
* Any cloud services or self- hosted edition of Red Hat OpenShift on versions 4.13-4.15.
```

File: docs/topics/mta-7-installing-web-console-on-openshift.adoc [hunk 4]
```diff
@@ -52,19 +52,19 @@ To successfully deploy, the {ProductShortName} Operator requires 3 RWO persisten
...
 * You must be logged in as a user with `cluster-admin` permissions.
 
 For more information, see link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles].
 
 .Procedure
 
-. In the OpenShift Container Platform web console, click *Operators → OperatorHub*.
+. In the OpenShift web console, click *Operators → OperatorHub*.
```
Comment (mpershina) [inline docs/topics/mta-7-installing-web-console-on-openshift.adoc:67]:
```suggestion
. In the Red Hat OpenShift web console, click *Operators → OperatorHub*.
```

File: docs/topics/mta-7-installing-web-console-on-openshift.adoc [hunk 5]
```diff
@@ -181,7 +181,7 @@ spec:
 . When prompted, create a new password.
 
 [id="installing-mta-operator-in-disconnected-environment_{context}"]
-== Installing the {ProductName} Operator in a disconnected {ocp-short} environment
+== Installing the {ProductName} Operator in a disconnected OpenShift environment
```
Comment (mpershina) [inline docs/topics/mta-7-installing-web-console-on-openshift.adoc:184]:
```suggestion
== Installing the {ProductName} Operator in a disconnected Red Hat OpenShift environment
```

File: docs/topics/mta-7-installing-web-console-on-openshift.adoc (hunk 1 above)
Comment (mpershina) [reply docs/topics/mta-7-installing-web-console-on-openshift.adoc:10]: @rromannissen, as for "this wording sounds weird", what exactly is incorrect in this case? From what I understand from the initial sentence version is that the MTA UI is installed when the user installs the MTA Operator. Is this wrong?

File: docs/topics/mta-7-installing-web-console-on-openshift.adoc (hunk 2 above)
Comment (mpershina) [reply docs/topics/mta-7-installing-web-console-on-openshift.adoc:55]: @rromannissen, the same here, does the sentence convey incorrect message? :-)

**Comment by mpershina** [comment]
Hi @rromannissen, tagging you in this MTA-2660-related PR. Let me know if you want to continue discussion here instead of the gdoc :-) You can see all of the suggested changes either already incorporated or as suggestions.

Thanks for your help!

**Comment by mpershina** [comment]
@rromannissen, please, double-check the changes and the preview :-) If you are ok with everything, I'll move the repsective Jira ticket to release pending.

File: docs/topics/mta-7-installing-web-console-on-openshift.adoc [hunk 6]
```diff
@@ -7,9 +7,11 @@
 
 = Installing the {ProductName} {WebName}
 
-You can install the {ProductName} ({ProductShortName}) {WebName} as part of the process of installing the {ProductShortName} Operator on the OpenShift Container Platform.
+You can install the {ProductName} ({ProductShortName}) {WebName} on all Red Hat OpenShift cloud services and Red Hat OpenShift self-managed editions.
 
-The {ProductShortName} Operator is a structural layer that manages resources deployed on Kubernetes (database, front end, back end) to automatically create an {ProductShortName} instance.
+IMPORTANT: To be able to create {ProductShortName} instances, you must first install the {ProductShortName} Operator.	
+
+The {ProductShortName} Operator is a structural layer that manages resources deployed on OpenShift (database, front end, back end) to automatically create an {ProductShortName} instance.
```
Comment (mportman12) [inline docs/topics/mta-7-installing-web-console-on-openshift.adoc:14]: Remove parentheses

File: docs/topics/mta-7-installing-web-console-on-openshift.adoc [hunk 7]
```diff
@@ -52,19 +54,19 @@ To successfully deploy, the {ProductShortName} Operator requires 3 RWO persisten
...
 == Installing the {ProductName} Operator and the {WebName}
 
-You can install the {ProductName} ({ProductShortName}) and the {WebName} on OpenShift Container Platform versions 4.13-4.15 when you install the {ProductName} Operator.
+You can install the {ProductName} ({ProductShortName}) and the {WebName} on Red Hat OpenShift versions 4.13-4.15.
 
 .Prerequisites
 
 * 4 vCPUs, 8 GiB RAM, and 40 GiB persistent storage.
-* OpenShift Container Platform 4.13-4.15 installed.
+* Any cloud services or self- hosted edition of Red Hat OpenShift on versions 4.13-4.15.
```
Comment (mportman12) [inline docs/topics/mta-7-installing-web-console-on-openshift.adoc:62]:
```suggestion
* Any cloud services or self-hosted edition of Red Hat OpenShift on versions 4.13-4.15.
```

File: docs/topics/mta-7-installing-web-console-on-openshift.adoc (hunk 6 above)
Comment (mpershina) [inline docs/topics/mta-7-installing-web-console-on-openshift.adoc:14]:
```suggestion
The {ProductShortName} Operator is a structural layer that manages resources deployed on OpenShift, such as database, front end, and back end, to automatically create an {ProductShortName} instance.
```

**Pull request merged**